projects/project-player-library/src/lib/pages/details-page/details-page.component.ts

```diff
@@ -2,7 +2,7 @@ import { Component, OnInit } from '@angular/core';
 import { MatDialog } from '@angular/material/dialog';
 import { Router } from '@angular/router';
 import { DailogPopupComponent } from '../../shared/dialog-popup/dailog-popup.component';
-import { projectDetailsData } from './project-details.component.spec.data';
+import { projectDetailsData } from '../../project-details.component.spec.data';
```

Comment:
Why is the path changed?

Reply:
it was coming from the details page which was in the page folder and it is overall data so i made it from parent folder